src/GovUk.Education.ExploreEducationStatistics.Admin/Mappings/MappingProfiles.cs

```diff
@@ -39,7 +39,13 @@ public MappingProfiles()
                     m => m.MapFrom(model =>
                         model.PublishScheduled.HasValue
                             ? model.PublishScheduled.Value.ConvertUtcToUkTimeZone()
-                            : (DateTime?) null));
+                            : (DateTime?) null))
+                .ForMember(dest => dest.LatestInternalReleaseNote,
+                    m => m.MapFrom(r =>
+                        r.ReleaseStatuses
+                            .OrderBy(rs => rs.Created)
+                            .LastOrDefault()
+                            .InternalReleaseNote));
```

Comment:
Does this work if there are no notes or do we create a new note with every new Release?

If there's no notes calling 'LastOrDefault' will be null so calling `InternalReleaseNote` will be a `NullReferenceException`

Reply:
I've stripped all the mapping out of MappingProfiles.cs and instead create a `LatestInternalReleaseNote` in Release.cs which handles getting the latest note from `ReleaseStatuses`